.circleci/heroku_deploy.sh

```diff
@@ -2,9 +2,11 @@
 
 if git remote | grep heroku > /dev/null; then
   git fetch heroku
-  git push --force heroku $CIRCLE_SHA1:master
+  DEPLOY_ENV=staging yarn deploy
```

Comment:
This winds up doing the asset stuff: minify/uglify/gzip/upload, as well as push to Heroku.